unlock-app/src/components/interface/checkout/alpha/Checkout/Captcha.tsx

```diff
@@ -83,7 +83,8 @@ export function Captcha({ injectedProvider, checkoutService }: Props) {
       id: 6,
       name: 'Solve captcha',
       to: 'CAPTCHA',
-      skip: !paywallConfig.captcha,
+      skip:
+        !paywallConfig.captcha || ['card', 'claim'].includes(payment.method),
```

Comment:
is not needed to show the captcha for credit card? what if `captcha=true` and payment method is `card` or 'claim'? it this case the captcha will not be visible

Reply:
Awesome! so LGTM!